go/multipooler/server/plugin_pq.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2025 Supabase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	_ "github.com/lib/pq" // PostgreSQL driver
```

Comment:
@sougou I spent some time trying to replace this with jackc/pgx and reverted that for a couple of reasons
- pgx has it's own pooling which we may not want given that we are going to implement our own
- there was a certain amount of translation code needed to convert from pgx results to proto messages. If this is anyway throw-away, it didn't seem worth writing that code.